src/commands/graph.ts

```diff
@@ -17,7 +17,13 @@ export function graphCommand() {
   const cwd = getCwd()
   if (!cwd) return
 
+  // Get the configured shell command
```

Comment:
Please remove that comment, as the fonction name below is explicit enough

Reply:
👍